frontend/src/components/camp/CampPicasso.vue

```diff
@@ -49,8 +51,8 @@ export default {
     periods () {
       return this.camp().periods()
     },
-    eventInstances () {
-      return this.periods.items.flatMap(period => period.event_instances())
+    events () {
+      return this.camp().events()
```

Comment:
Wo wird das verwendet?

Reply:
Nur im `v-if="events._meta.loading"` auf dem skeleton loader. Aber der Nebeneffekt davon ist, dass der events Endpoint abgerufen wird. In jedem Event in dieser Liste sind die EventInstances embedded, darum werden so auf einen Schlag (mit einem Request) alle EventInstances des Camps im Store verfügbar.
Erst wenn das gemacht ist wird der Picasso dann angezeigt. Der `period.event_instances().items` lädt zwar noch vorher die Liste mit Links zu EventInstances, aber die Inhalte sind dann alle schon im Store und werden nicht einzeln gefetcht.
Könnte man in Zukunft vielleicht noch verbessern indem man nicht einmal auf die Periods wartet, sondern skeleton-loaders anzeigt während `this.periods._meta.loading || this.events._meta.loading`.